nagios-plugins-xrootd.spec

```diff
@@ -0,0 +1,55 @@
+# Package needs to stay arch specific (due to nagios plugins location), but
+# there's nothing to extract debuginfo from
+%global debug_package %{nil}
+
+%define nagios_plugins_dir %{_libdir}/nagios/plugins
+
+Name:       nagios-plugins-xrootd
+Version:    0.0.1
+Release:    1%{?dist}
+Summary:    Nagios probes to be run remotely against XRootD endpoints
+License:    ASL 2.0
```

Comment:
The license of this repo is currently MIT

Reply:
fixed